src/shared/widgets/floating/static-doorhanger.jsx

```diff
@@ -16,13 +16,17 @@ import CSSModules from 'react-css-modules';
 import WidgetComponent from '../helpers/widget-component';
 import Styles from './static-doorhanger.css';
 import Tooltip from './tooltip';
-import CSSArrow from '../css-arrow';
```

Comment:
I'd prefer to remove it since it's not used anywhere.. we can find it later on if we end up needing it